public/video-ui/src/components/VideoPage/VideoPage.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import Q from 'q';
```

Comment:
I saw that Q was being used in the PandaReqwest service and I don't think Promise is being used here, but I replace Q with Promise if you'd like